docs/cli/pyats_logs.rst

```diff
@@ -87,3 +87,43 @@ The following is an example of what the webpage looks like:
 
 Each tab shows the detailed results of each section of each testscript, as well
 as all commands executed and specific sections of logs.
+
+
+
+pyats logs upload
+---------------
+
+This subcommand will upload a zipped file to Taas Logviewer. It is an internal to 
```

Comment:
This command upload a .zip pyATS Archive to TaaS Logviewer.

Reply:
fixed